### DISCUSSION TITLE
Who can tell me what is the default username and password

### DISCUSSION THREAD

**Discussion by ferperji** [johang/sd-card-images#112] [Q&A]
hi,all
I first use the johang SD-CARD-IMAGES for my Tinker Board S and Rock Pi 4C Plus.
I don`t konw default username and password,for example i use linke "https://dl.sd-card-images.johang.se/debians/2024-02-05/ubuntu-noble-armhf-thoo9f.bin.gz",Is the Password thoo9f ? and what is the username ?
Who can tell me ? 
Thanks .

**Comment by jfargen** [comment]
If you look at the download page it should have a statement like this  "When the dd command is done, put the SD card in your board and power it on. The board will boot up and assign itself an IP address using DHCP. The last part of the Debian/Ubuntu image filename is the default **root** password ("thoo9f" for this image)." 

The user **root**, is commonly used as the administrator user with the highest possible permissions, in linux/unix systems.
